CHANGELOG.markdown

```diff
@@ -16,6 +16,7 @@
   functions in `Control.Lens.TH`.
 * Add `Data.HashSet.Lens.hashMap`, an `Iso` between a `HashSet a` and a
   `HashMap a ()`.
+* Allow building with `transformers-0.6.*` and `mtl-2.3.*`.
```

Comment:
This could mention what this means for `ListT` and `ErrorT` instances, to clarify that you may have configuration with `ListT` available in transformers, but without `Zoom` instance. I.e. the same asked/answered on the issue.

Reply:
Good idea. I've added a follow-up to this changelog entry to explain the subtleties of `ListT`, `ErrorT`, and their `Zoom` instances.